change/@ni-eslint-config-angular-9ad7619f-6545-4755-83fa-124156f09929.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
FYI I renamed #134 to track the work of evaluating new rules that are not yet enabled and I copied the list from this PR's description into a new comment there.